transformer_lm/main.py

```diff
@@ -67,7 +79,9 @@ def main(args):
     vocab, train, valid, test = datasets.load_dataset(args.dataset)
 
     # Initialize model:
-    model = TransformerLM(len(vocab), args.num_blocks, args.dim, args.num_heads)
+    model = TransformerLM(
+        len(vocab), args.num_blocks, args.dim, args.num_heads, args.checkpoint
+    )
```

Comment:
Should we also do a `del grads` below? Seems like it could be good to include that as a best practice.